test/cri-containerd/main.go

```diff
@@ -26,20 +26,25 @@ import (
 )
 
 const (
-	daemonAddress                     = "tcp://127.0.0.1:2376"
-	connectTimeout                    = time.Second * 10
-	testNamespace                     = "cri-containerd-test"
+	daemonAddress  = "tcp://127.0.0.1:2376"
+	connectTimeout = time.Second * 10
+	testNamespace  = "cri-containerd-test"
+
 	wcowProcessRuntimeHandler         = "runhcs-wcow-process"
 	wcowHypervisorRuntimeHandler      = "runhcs-wcow-hypervisor"
 	wcowHypervisor17763RuntimeHandler = "runhcs-wcow-hypervisor-17763"
 	wcowHypervisor18362RuntimeHandler = "runhcs-wcow-hypervisor-18362"
-	lcowRuntimeHandler                = "runhcs-lcow"
-	imageLcowK8sPause                 = "k8s.gcr.io/pause:3.1"
-	imageLcowAlpine                   = "docker.io/library/alpine:latest"
-	imageLcowCosmos                   = "cosmosarno/spark-master:2.4.1_2019-04-18_8e864ce"
-	testGPUBootFiles                  = "C:\\ContainerPlat\\LinuxBootFiles\\nvidiagpu"
-	alpineAspNet                      = "mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11"
-	alpineAspnetUpgrade               = "mcr.microsoft.com/dotnet/core/aspnet:3.1.2-alpine3.11"
+	testDeviceUtilFilePath            = "C:\\ContainerPlat\\device-util.exe"
+	testDriversPath                   = "C:\\ContainerPlat\\testdrivers"
+
```

Comment:
nit: newline

Reply:
I added the newline on purpose :) to separate the lcow and wcow passed variables.

Reply:
Ahh. It's growing on me not going to lie. We can leave 😄